CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ parent: README
 nav_order: 1
 ---# Changelog
 
+## [2.1.16] - 2021-07-01
+- Add `delete_retention_ms` to `aiven_kafka_topic` resource read method
+- Add `use_source_project_billing_group` support for `aiven_project` resource
+- Add service integration `endpoint_id` validation
+- Add vpc peering connection `state_info` filed type conversion
```

Comment:
`filed` => `field`. Also could write VPC in all caps

Reply:
done